mne/viz/tests/test_3d.py

```diff
@@ -101,3 +103,16 @@ def test_plot_trans():
     assert_raises(ValueError, plot_trans, evoked.info, trans_fname,
                   subject='sample', subjects_dir=subjects_dir,
                   ch_type='bad-chtype')
+
+
+@testing.requires_testing_data
+@requires_mayavi
+def test_plot_dipoles():
+    """Test plotting dipoles
+    """
+    fname_dip = op.join(data_dir, 'MEG', 'sample',
+                        'sample_audvis_trunc_set1.dip')
+    dipoles = read_dipole(fname_dip)
+    coord_trans = invert_transform(read_trans(trans_fname))
+    coord_trans = coord_trans['trans']
```

Comment:
the invert transform should be done in the plot function
if needed. You should not expect the mne users to understand
this.